README.md

```diff
@@ -87,6 +87,12 @@ You can run a specific test case by setting the `TESTFLAGS` variable.
 # make test TESTFLAGS="-run=SomeTestFunction"
 ```
 
+You can run a test in your proxy environment by setting `DOCKER_BUILD_PROXY` and `DOCKER_RUN_PROXY` variables.
+
+```bash
+# make test DOCKER_BUILD_PROXY="--build-arg HTTP_PROXY=http://yourproxy/" DOCKER_RUN_PROXY="-e HTTP_PROXY=http://yourproxy/"
```

Comment:
Should not these be `DOCKER_BUILD_FLAGS` and `DOCKER_RUN_FLAGS`?

Reply:
@AkihiroSuda Thank you for your comments. I thought these variables should be only used for proxy settings and named these variables.  But either is fine.